scripts/fetch-brew-bottle.sh

```diff
@@ -35,6 +35,8 @@ fi
 BOTTLE_NAME="${1}"
 BOTTLE_PATH="/tmp/${BOTTLE_NAME}.tar.gz"
 
+trap "echo 'Failed to download bottle. Check your authentication token and network connection." EXIT ERR INT QUIT
```

Comment:
If you're adding a `trap` for errors, I'd move it above function definitions, since those can fail too.

Reply:
It is above the calls to those functions, isn't it enough?